source/game/field/ObjectCollisionSphere.cc

```diff
@@ -0,0 +1,17 @@
+#include "ObjectCollisionSphere.hh"
+
+namespace Field {
+
+/// @addr{0x808368D0}
+ObjectCollisionSphere::ObjectCollisionSphere(f32 radius, const EGG::Vector3f &center)
```

Comment:
```suggestion
ObjectCollisionSphere::ObjectCollisionSphere(const EGG::Sphere3f &sphere)
```
Bad idea? We have to split out the components in the body anyway...

Reply:
It's provably not a `Sphere3f` in the base game, so I'm hesitant to include that here.

I might consider grouping the sphere/cylinder components together into respective structs later on, but I don't think such a thing is necessary at the moment.